README.md

```diff
@@ -41,3 +41,13 @@ Copyright © 2015-2020 Hugo Locurcio and contributors
 Unless otherwise specified, code is licensed under the MIT license.
 Media and content are licensed under
 [CC BY-SA 3.0 Unported](https://creativecommons.org/licenses/by-sa/3.0/).
+
+Screenshots from:
+
+- mtg.jpg and mtg_sea.jpg, Minetest Game, CC BY-SA 3.0
+- rpg16.jpg, RPG16 texture pack (Hugues Ross), CC BY-SA 4.0
+- soothing32.jpg, Soothing32 (Zughy), CC BY-SA 4.0
+- i3.jpg, i3 (Jean-Patrick Guerrero), CC BY-SA 3.0
+- mineclone2.jpg, Mineclone2 (Wuzzy), GPL 3.0.
```

Comment:
GPL is a code license

do media licenses even apply to in-game screenshots? isn't that fair use?
if so then they shouldn't be listed at all